oci.go

```diff
@@ -58,7 +57,12 @@ func containerExists(containerID string) (bool, error) {
 func validCreateParams(containerID, bundlePath string) error {
 	// container ID MUST be provided.
 	if containerID == "" {
-		return fmt.Errorf("Missing container ID")
+		return errNeedContainerID
```

Comment:
Here again, containerExists should fail with a nice error message including the containerId instead of special casing the "" value.